### PR TITLE
feat: add dockerfile.arm64

### PR DESCRIPTION
**Pull request by hyy0322**
/cc @supereagle @zhujian7

### CODE REVIEW

File: build/Dockerfile.arm64 [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM arm64v8/golang:1.12-alpine as base
+ADD .   /go/src/github.com/coreos/clair/
+WORKDIR /go/src/github.com/coreos/clair/
+RUN apk add --no-cache git rpm xz build-base
+RUN export CLAIR_VERSION=$(git describe --always --tags --dirty) \
+    && go install -ldflags "-X github.com/coreos/clair/pkg/version.Version=$CLAIR_VERSION" -v github.com/coreos/clair/cmd/clair
+
+FROM arm64v8/alpine:3.8
+LABEL maintainer="chende@caicloud.io"
```
Comment (supereagle) [inline build/Dockerfile.arm64]: Update maintainer

Comment (hyy0322) [reply build/Dockerfile.arm64]: Done.

**Pull request merged**